src/apps/dictionaries/utils.ts

```diff
@@ -32,3 +32,17 @@ export const dictionaryNameFromUrl = (url: string): string => {
   let words = url.split("/");
   return words[words.length - 2];
 };
+
+export const getDictionaryTypeFromPreviousPath = (previousPath: String) => {
```

Comment:
Nice one!

Reply:
Thanks @hadijahkyampeire. Let me delete the comment and push